<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDiag</class>
 <widget class="QDialog" name="LoginDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>280</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nest</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="LoginBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 13pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:hover
{
font: 13pt &quot;华文楷体&quot;;
color:rgb(117, 117, 117);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:pressed
{
font: 13pt &quot;华文楷体&quot;;
color:rgb(182, 182, 182)
border:2px groove gray;border-radius:10px;padding:2px 4px
}
</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 13pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:hover
{
font: 13pt &quot;华文楷体&quot;;
color:rgb(117, 117, 117);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:pressed
{
font: 13pt &quot;华文楷体&quot;;
color:rgb(182, 182, 182)
border:2px groove gray;border-radius:10px;padding:2px 4px
}
</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文楷体&quot;;
</string>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文楷体&quot;;
</string>
   </property>
   <property name="text">
    <string> 密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usrLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 12pt &quot;Calibri&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 12pt &quot;Calibri&quot;;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Tip">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string> 请输入密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RegBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 13pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:hover
{
font: 13pt &quot;华文楷体&quot;;
color:rgb(117, 117, 117);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:pressed
{
font: 13pt &quot;华文楷体&quot;;
color:rgb(182, 182, 182)
border:2px groove gray;border-radius:10px;padding:2px 4px
}
</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;
</string>
   </property>
   <item>
    <property name="text">
     <string> 酒店</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string> 客户</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>管理员</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string> 经理</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文楷体&quot;;
</string>
   </property>
   <property name="text">
    <string>用户类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文楷体&quot;;
</string>
   </property>
   <property name="text">
    <string>欢迎使用本系统 :D</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
